src/utils/signerUtils.ts

```diff
@@ -0,0 +1,62 @@
+import store from '@/store'
+import getEnvApi from '@/utils/get-env-api'
+
+type SmartContractRequestParams = {
+  appAddress: string
+  provider?: string
+  forwarderAddress: string
+  gateway: string
+  accessToken: string
+  method?: string
+  value?: string
+}
+
+let signTransaction: (
+  method: string,
+  value?: globalThis.SmartContractAcceptedValue
+) => Promise<string>
+
+function getTxRequestProps(): SmartContractRequestParams {
```

Comment:
Prefer either consistently using 'tx' everywhere to refer to transactions or expand to 'transaction'.

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/118/commits/dae7657683180d804c259a7d845ec3cf3f185f36